oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java

```diff
@@ -44,6 +44,9 @@ public BanyanDBIndexInstaller(Client client, ModuleManager moduleManager, Banyan
 
     @Override
     public boolean isExists(Model model) throws StorageException {
+        if (!model.isTimeSeries()) { // UITemplate
+            return true;
+        }
```

Comment:
Why do we highlight this? The not-timer-series model would not be created in `#createTable`.

Reply:
OK, I just noticed https://github.com/apache/skywalking/issues/9965.

I think we need to update the title a little. The correct one as title and change log should be 

`Fix not-time-series model blocking OAP boots in no-init mode`.

```suggestion
        if (!model.isTimeSeries()) {
            return true;
        }
```

This comment is not necessary, we wouldn't remember to update this when we add next new not-time-series model.

Reply:
Follow this line https://github.com/apache/skywalking/blob/master/oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java#L90

Do I have to keep it or remove them at all?

Reply:
Remove this one too.